### ISSUE TITLE
pod  pgo error:  no matches for kind "CronJob" in version "batch/v1beta1"

### ISSUE THREAD

**Issue by itswl** [CrunchyData/postgres-operator-examples#164]
``` 
# kubectl get pod -n postgres-operator
NAME                           READY   STATUS             RESTARTS          AGE
pgo-55c9468d65-4x44t           0/1     CrashLoopBackOff   119 (4m46s ago)   10h
```

```
# kubectl logs -n postgres-operator pgo-55c9468d65-4x44t
time="2022-12-02T01:58:24Z" level=info msg="{\"pgo_versions\":[{\"tag\":\"v5.1.0\"},{\"tag\":\"v5.0.5\"},{\"tag\":\"v5.0.4\"},{\"tag\":\"v5.0.3\"},{\"tag\":\"v5.0.2\"},{\"tag\":\"v5.0.1\"},{\"tag\":\"v5.0.0\"}]}" X-Crunchy-Client-Metadata="{\"deployment_id\":\"dc03e8a6-1191-42fa-990f-3c6b59e5f9d9\",\"kubernetes_env\":\"v1.25.3\",\"pgo_clusters_total\":0,\"pgo_version\":\"5.2.1-0\",\"is_open_shift\":false}" file="internal/upgradecheck/http.go:181" func=upgradecheck.CheckForUpgradesScheduler version=5.2.1-0
time="2022-12-02T01:58:24Z" level=error msg="if kind is a CRD, it should be installed before calling Start" error="no matches for kind \"CronJob\" in version \"batch/v1beta1\"" file="sigs.k8s.io/controller-runtime@v0.8.3/pkg/log/deleg.go:144" func="log.(*DelegatingLogger).Error" kind=CronJob.batch version=5.2.1-0
panic: no matches for kind "CronJob" in version "batch/v1beta1"

goroutine 1 [running]:
main.assertNoError(...)
        github.com/crunchydata/postgres-operator/cmd/postgres-operator/main.go:41
main.main()
        github.com/crunchydata/postgres-operator/cmd/postgres-operator/main.go:105 +0x56d

```

```
kubectl version --short
Flag --short has been deprecated, and will be removed in the future. The --short output will become the default.
Client Version: v1.25.3
Kustomize Version: v4.5.7
Server Version: v1.25.3
```

```
# kubectl api-resources
NAME                SHORTNAMES        APIVERSION                         NAMESPACED   KIND
cronjobs                          cj                batch/v1                                    true         CronJob
```

**Comment by whs-dot-hk** [comment]
https://github.com/CrunchyData/postgres-operator/issues/3365

**Comment by benjaminjb** [comment]
Hello, we recently released PGO 5.3.0, which is updated to support K8s 1.25. That should fix this problem and in the future, https://github.com/CrunchyData/postgres-operator#supported-platforms will let you know what K8s versions are compatible with PGO.
